klippy/extras/gcode_macro.py

```diff
@@ -37,6 +37,8 @@ def cmd(self, params):
             self.gcode.run_script_from_command(script)
         finally:
             self.in_script = False
+    def contains(self, gc):
+        return gc.upper() in self.script.upper()
```

Comment:
I think this change should no longer be necessary.

Reply:
Yes, I will remove it.